tests/pds/dataservice_test.go

```diff
@@ -133,24 +134,28 @@ var _ = Describe("{ValidatePDSHealthInCaseOfFailures}", func() {
 				wg.Add(2)
 				go func() {
 					defer wg.Done()
-					log.InfoD("Deleting the data service pods")
-					err = pdslib.DeletePods(pdsPods)
-					log.FailOnError(err, "Error while deleting pods")
+					log.InfoD("Deleting the first data service pod %s", pdsPods[0].GetName())
+					err = k8sCore.DeletePod(pdsPods[0].GetName(), params.InfraToTest.Namespace, true)
+					log.FailOnError(err, "Error while deleting pod")
 				}()
 
 				go func() {
 					defer wg.Done()
 					log.InfoD("Validating the data service pod status in PDS Control Plane")
-					err = pdslib.ValidatePDSDeploymentStatus(deployment, "Down", 5*time.Second, 30*time.Minute)
+					pdsHealth.Degraded = "Degraded"
+					pdsHealth.Down = "Down"
+					err = pdslib.ValidatePDSDeploymentDowntime(deployment, pdsHealth, 5*time.Second, 30*time.Minute)
```

Comment:
Can we use constants here like : timeOut and timeInterval

Reply:
done